CMakeLists.txt

```diff
@@ -156,17 +167,51 @@ if(NGEN_WITH_NETCDF)
     add_compile_definitions(NETCDF_ACTIVE)
 endif()
 
+if(NGEN_WITH_EXTERN_CXX_MODELS)
+    # This is set because SLOTH will link to gtest by default,
+    # which causes a configure error.
+    set(PACKAGE_TESTS OFF)
```

Comment:
I'm kinda bothered that we end up setting this 'globally' to handle one package being weird. Scoping it to a `function` or an `include()`'d file would ameliorate that, ugly as it would be.